piper/src/main/resources/workflows/Exome.sh

```diff
@@ -83,7 +87,7 @@ fi
 #---------------------------------------------
 # Run the qscript
 #---------------------------------------------
-piper -S ${SCRIPTS_DIR}/DNABestPracticeVariantCalling.scala \
+piper -S ${_THIS_SCRIPT_LOCATION}/../qscripts/DNABestPracticeVariantCalling.scala \
 	      --xml_input ${PIPELINE_SETUP} \
 	      --isExome \
 	      --gatk_interval_file ${INTERVALS} \
```

Comment:
to be consistent with the other workflows, should it be `--global_config ${_THIS_SCRIPT_LOCATION}/../conf/uppmax_global_config.xml` below?